sql/src/test/java/io/crate/expression/scalar/string/StringLeftRightFunctionTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.expression.scalar.string;
+
+import io.crate.expression.scalar.AbstractScalarFunctionsTest;
+import io.crate.expression.symbol.Literal;
+import io.crate.types.DataTypes;
+import org.junit.Test;
+
+public class StringLeftRightFunctionTest extends AbstractScalarFunctionsTest {
+
+    @Test
+    public void testEvaluateLeftFunc() throws Exception {
+        Literal<Object> value = Literal.of(DataTypes.STRING, "crate.io");
+        assertEvaluate("left(name, 10)", null, Literal.of((String) null));
+        assertEvaluate("left(name, null)", null, value);
+        assertEvaluate("left(name, 0)", "", value);
+        assertEvaluate("left(name, 100)", "", Literal.of(""));
+        assertEvaluate("left(name, 5)", "crate", value);
+        assertEvaluate("left(name, -3)", "crate", value);
```

Comment:
minor nitpick: maybe adding one test where the offset is a column does make sense.

Reply:
ok, will have a look.

Reply:
addressed